AutocompleteClient/FW/Logic/Query/CIOQueryFmtOptions.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  CIOQueryFmtOptions.swift
+//  AutocompleteClient
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import Foundation
+
+public typealias FmtOptions = (key: String, value: String)
```

Comment:
Should this be `FmtOption` (singular)? Same goes for the places where it's used below

Reply:
yep